src/App.tsx

```diff
@@ -2,10 +2,20 @@ import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
-import { Movie } from './types/Movie';
+import { MovieData } from './types/MovieData';
 
-export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+export const App: React.FC = () => {
+  const [movies, setMovie] = useState<(MovieData | null)[]>([]);
```

Comment:
<img width="367" alt="image" src="https://user-images.githubusercontent.com/66749919/205448968-95a7e059-3b78-40c5-86ae-2a7a51ddf03f.png">
you need to add Movie type and normalize MovieData to this type